tex/gregoriotex-signs.tex

```diff
@@ -48,7 +48,11 @@
 % To remedy that, we define \gre@hskip to be \hskip outside a discretionary, and
 % \kern inside a discretionary. This is what these macros do:
 
-\def\gre@falsepenalty#1{}%
+% save asked penalty in the case where we don't really use it (discretionary, etc.)
+\xdef\gre@savedpenalty{0}
+
+% gre@falsepenalty just saves the last seen penalty
+\def\gre@falsepenalty#1{\xdef\gre@savedpenalty{#1}}%
```

Comment:
Sure (you can push in the branch if you want), but this kind of details could be done in an ironing pass after rc1